psalm-baseline.xml

```diff
@@ -1970,6 +1964,9 @@
     <PropertyNotSetInConstructor occurrences="1">
       <code>MultiTableUpdateExecutor</code>
     </PropertyNotSetInConstructor>
+    <PropertyTypeCoercion occurrences="1">
+      <code>$this-&gt;_sqlStatements</code>
+    </PropertyTypeCoercion>
```

Comment:
The code is hard to analyse statically, but from my human point of view, what is intended here is a `list`

Reply:
Agree. `mixed[]` doesn't fit for _sqlStatements.